Cargo.toml

```diff
@@ -25,7 +25,7 @@ targets = [
 ]
 
 [dependencies]
-libc = { git = "https://github.com/rust-lang/libc", rev = "9c1489fa8", features = [ "extra_traits" ] }
+libc = { git = "https://github.com/rust-lang/libc", features = [ "extra_traits" ] }
```

Comment:
It's good to always specific a `rev` when using git dependencies with Cargo.  Otherwise, you force Cargo to fetch the repository each and every time.

Reply:
Alright, fixed!